### PR TITLE
title : exception handling in c#

### PR DESCRIPTION
**Pull request by hemant404**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] Checked for all grammatical errors in the blog.
- [x] Checked for any duplicate blog entry.

##### Description of change

Added a new blog on Exception Handling in C#

##### Author Details for Blog Post Entry

Id -  Hemant Manwani

### CODE REVIEW

File: content/blog/exception_handling_in_c#/index.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+**System.InvalidCastException** – Occurs when you try to cast an object to a type that it can’t be cast to
+
+## Exception Handling
+
+The process of handling any run time error is called exception handling. The exception handling is performed to maintain the normal flow of the application. 
+
+Exception handling is like
+
+![Exception Handling](Exception_Handling.gif)
```
Comment (ashish8947) [inline content/blog/exception_handling_in_c#/index.md:42]: please use .png image

Comment (hemant404) [reply content/blog/exception_handling_in_c#/index.md]: @ashish8947  Removed gif file.

**Review by pareek-narendra — COMMENTED** [review]
Cover Image should be in 3:2

**Review by pareek-narendra — COMMENTED** [review]
Looks Good

**Pull request merged**